cogs/stats.py

```diff
@@ -0,0 +1,42 @@
+import discord
+from discord.ext import commands
+import pylunar
```

Comment:
Does pylunar need to be installed as a package?
If so add to a requirements.txt